libraries/AP_HAL_PX4/RCOutput_Tap.cpp

```diff
@@ -0,0 +1,567 @@
+/****************************************************************************
```

Comment:
Not sure... Initial code wasn't mine so I cannot relicense it. Initially it would be plain copy and paste but in the end it ended up being different. Not sure what to do here.